src/github.com/twitchyliquid64/CNC/util/functions.go

```diff
@@ -0,0 +1,22 @@
+package util
+
+import (
+  "github.com/twitchyliquid64/CNC/logging"
+  "github.com/robertkrimen/otto"
+)
+
+func GetFunc(argument otto.Value, vm *otto.Otto) otto.Value {
+  if (argument.IsFunction()) {
+    return argument;
+  }
+
+  mname := argument.String()
+  logging.Warning("Using a string to name parameters is now depricated. Please use callbacks directly: " + mname)
```

Comment:
logging.Warning takes two parameters: also fix spelling

 try this:

logging.Warning("plugin-builtin", "Using a string to name parameters is now deprecated. Please use callbacks directly: " + mname)